File: unit-3/sprint-2/evaluation/firstapp/app/src/main/java/com/example/firstapp/APIService.java
package com.example.firstapp;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface APIService {

    @GET("shivarajp/2cbe00030c04472c9d8ad4b0ec112dbe/raw/c651391e428182f08d60d59e79073f3fcf79b858/nobroker")
    Call<List<ResponseModel>> getData(@Path("") int id);
}

File: unit-3/sprint-4/day-2/you/18AugPostRequest/app/src/main/java/com/example/a18augpostrequest_kotlin/ApiService.kt
package com.example.a18augpostrequest_kotlin

import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.GET
import retrofit2.http.POST

interface ApiService {

    @POST("posts")
    fun postData(@Body requestModel: RequestModel): Call<ResponseModel>

//    @GET("posts")
//    fun getPost(): Call<ResponseModel>

}